chainer_chemistry/datasets/molnet/molnet_config.py

```diff
@@ -0,0 +1,282 @@
+import chainer.functions as F
+from chainer_chemistry.datasets.molnet.chembl_tasks import chembl_tasks
+from chainer_chemistry.datasets.molnet.toxcast_tasks import toxcast_tasks
+
+molnet_base = 'http://deepchem.io.s3-website-us-west-1.amazonaws.com/datasets/'
+
```

Comment:
It would be nice for users if there is comment or doc to explain each key's meaning.
dict key shows `dataset_type` and "dataset_type" shows ..., "url" shows ... etc...
Especially, "tasks" represents labels in chainer chemistry.

Reply:
we can do in next PR as well.

Reply:
I agree. Additionally I would like to write documents about it somewhere.